src-vue/src/plugins/store.ts

```diff
@@ -382,6 +388,16 @@ async function _initializeApp(state: any) {
         // Check installed Northstar version if found
         await _get_northstar_version_number(state);
     }
+
+    await invoke<[number, number]>("get_server_player_count")
```

Comment:
Didn't we speak about pinging master server every 5 minutes?

Reply:
Eh, I'll do it later, wanna get that release out :P